src/main/java/ru/itis/services/ItemService.java

```diff
@@ -21,14 +21,14 @@ public ItemService(ItemsRepository itemsRepository, WishListService wishListServ
         this.wishListService = wishListService;
     }
 
-    public void addNewItem(ItemForm itemForm, Long listId) {
+    public void addNewItem(ItemDto itemDto, Long listId) {
         WishList wishList = getWishList(listId);
         Item newItem = Item.builder()
-                .name(itemForm.getName())
-                .price(itemForm.getPrice())
-                .link(itemForm.getLink())
+                .name(itemDto.getName())
```

Comment:
Too many params. Builder is an antipattern that hides ugly constructor call from you.
If you don't use builder, you will see that your real code
```java
Item newItem = new Item(itemDto.getName(), itemDto.getPrice(), itemDto.getLink(), wishList, itemDto.getDescription())
```
is a mess.

Reply:
fixed